### PR TITLE
Windows platform page updates.

### LINKED ISSUE #601: Update windows platform page
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)

## Expected Behavior
Windows platform page needs a few updates to make it easier to at least get up and running:
- [x] Call out file extensions based on your editor saving `.json` files as some other file type.
- [x] Move _DANGER to _WARNING
- [ ] Distinguish when to use `PowerShell` or `cmd` (or just use `cmd` if possible)

## Current Behavior

While seasoned windows users might be able to get through the platform guide without issue, the listed items should at least be called out to cover newer users.

## Possible Solution
Add some updates.

## Screenshots (if appropriate):

screen shot 2018-07-24 at 9 24 37 pm

### PR DESCRIPTION
**Pull request by rgeniesse**
## Description
Closes #601.

## Motivation and Context
As a non-Windows user, the page needed some additional details to help move other non-Windows users along. Notes and warnings were also put onto new lines to help with future changes and better git diffs.

## How Has This Been Tested?
Locally with yarn.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New doc/guide
- [ ] Fixing errata
- [X] Update (Add missing or refresh existing content)

## Checklist:

- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] All tests have passed.

### CODE REVIEW

File: content/sensu-core/0.29/platforms/sensu-on-microsoft-windows.md [hunk 1]
```diff
@@ -34,10 +34,8 @@ installer package (i.e. a .msi file), which is available for download from the
 
 ### Download and install Sensu using the Sensu MSI {#download-and-install-sensu-core}
 
-_NOTE: As of Sensu version 1.0 repository URLs have changed.  To
-install or upgrade to the latest version of Sensu, please ensure you
-have updated existing configurations to follow the repository URL
-format specified below._
+_NOTE: As of Sensu version 1.0 repository URLs have changed.
```
Comment (cwjohnston) [inline content/sensu-core/0.29/platforms/sensu-on-microsoft-windows.md]: Let's say "as of version 0.29" in this document, use version 1.0 in all later versions

File: content/sensu-core/0.29/platforms/sensu-on-microsoft-windows.md [hunk 2]
```diff
@@ -156,8 +160,7 @@ file path to `C:\opt\sensu\config.json`, the Sensu configuration directory to
 
 ### Install the Sensu client Windows service
 
-Open a Command Prompt and use the [Windows SC][10] utility to create the Windows
-service for the Sensu client:
+Open an Administrative Command Prompt and use the [Windows SC][10] utility to create the Windows service for the Sensu client:
```
Comment (cwjohnston) [inline content/sensu-core/0.29/platforms/sensu-on-microsoft-windows.md:163]: :+1:

File: content/sensu-core/0.29/platforms/sensu-on-microsoft-windows.md [hunk 3]
```diff
@@ -64,13 +66,15 @@ configuration from the following locations:
 - `C:\opt\sensu\config.json`
 - `C:\opt\sensu\conf.d\`
 
+_WARNING: Ensure that the configure files you create have a `.json` file extension and not, for example, a `.json.txt` file extension.
```
Comment (cwjohnston) [inline content/sensu-core/0.29/platforms/sensu-on-microsoft-windows.md:69]: 👍

**Review by cwjohnston — CHANGES_REQUESTED** [review]
Not your mistake but one change needed here. Otherwise looks good. 👍

**Review by apaskulin — APPROVED** [review]
Looks good! 🌟

**Pull request merged**